account_invoice_import_tensorflow/models/account_invoice_import.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import json
+import requests
+from odoo import models
+from odoo.tools import float_round
+
+
+class AccountInvoiceImport(models.TransientModel):
+    _inherit = "account.invoice.import"
+
+    def fallback_parse_pdf_invoice(self, file_data):
+        parsed_data = self._account_invoice_import_tensorflow_parse(file_data)
+        if parsed_data.get('failed'):
+            return super(AccountInvoiceImport, self).fallback_parse_pdf_invoice(file_data)
+        return parsed_data
+
+    def _account_invoice_import_tensorflow_parse(self, file_data):
+        """Send file to serving container and return its data"""
+        #response = requests.post(
+        #    self.env['ir.config_parameter'].get_param(
+        #        'account_invoice_import_tensorflow.predict_url'
+        #    ),
+        #    json=self._account_invoice_import_tensorflow_predict_data(
+        #        file_data
+        #    ),
+        #)
+        # return self._account_invoice_import_tensorflow_parse_response(response.json())
+        response_data = dict(predictions=json.loads(file_data))
+        if not response_data['predictions']:
+            # TODO how does a response of a failed prediction look?
```

Comment:
We can make sure that the model always outputs a JSON file with all fields, if for some reason the model prediction fails (e.g. due to corrupt input), we can catch the error and return a JSON file with empty predictions: "vendor_id": "", "vendor_name": "", etc.